src/main/java/com/google/api/codegen/transformer/go/GoGapicSurfaceTransformer.java

```diff
@@ -338,7 +338,11 @@ void addXApiImports(InterfaceContext context, Collection<MethodModel> methods) {
   void addXExampleImports(InterfaceContext context, Iterable<? extends MethodModel> methods) {
     ImportTypeTable typeTable = context.getImportTypeTable();
     typeTable.saveNicknameFor("context;;;");
-    typeTable.saveNicknameFor(context.getProductConfig().getPackageName() + ";;;");
+    typeTable.saveNicknameFor(
+        context.getProductConfig().getPackageName()
```

Comment:
Woah, how does this change, which includes semicolons, result in the baseline change? Are the semicolon chars just part of the internal representation of the import type?

Reply:
> Are the semicolon chars just part of the internal representation of the import type

That's my guess as well. I don't know 100%, I'd let @pongad confirm.

The `gapic-generator-go`, expects the gapic pkg path to be in a similar the form `path/to/impot;alias` which is then split into the path & alias name.